.drone.yml

```diff
@@ -11,6 +11,11 @@ services:
     image: percona/percona-server:5.6
     environment:
       - MYSQL_ALLOW_EMPTY_PASSWORD=yes
+  selenium:
+    image: selenium/standalone-chrome:3.11
```

Comment:
Why are we locking selenium at 3.11? Could/should be 3.

Reply:
https://github.com/search?q=org%3Aopeneuropa+image%3A+selenium%2Fstandalone-chrome-debug&type=Code
We don't seem to have a consensus at the moment :)

Reply:
In absence of a consensus, the latest version is typically preferred no?